upload-s3/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Upload S3'
+description: 'Uploads package to S3'
+inputs:
+  bucket:
+    required: true
+  product:
+    required: true
+  branch:
+    required: true
+  build:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create and release
+      run: aws s3api put-object --bucket ${{ inputs.bucket }} --key ${{ inputs.product }}/${{ inputs.branch }}/${{ inputs.build }}/release.tar.gz --body release.tar.gz
+      shell: bash
+    - name: Upload installer features
+      run: aws s3api put-object --bucket ${{ inputs.bucket }} --key ${{ inputs.product }}/${{ inputs.branch }}/${{ inputs.build }}/install/installer-features.txt --body installer-features.txt
+      shell: bash
+    - name: Update latest build
+      run: aws s3api put-object --bucket ${{ inputs.bucket }} --key ${{ inputs.product }}/${{ inputs.branch }}/latestBuild --body latestBuild
+      shell: bash

```

Comment:
`Nit`
Does shell have to be redundantly specified at each step? Could it be globally declared?

Reply:
I think each step needs its own `shell` line

Reply:
Each step could run multiline bash (at least in workflows you can), but not sure if that's something you would want to do. E.g.

```yaml
    - name: Upload installer features
      run: |
        set -eu
        aws s3api put-object --bucket ${{ inputs.bucket }} --key ${{ inputs.product }}/${{ inputs.branch }}/${{ inputs.build }}/install/installer-features.txt --body installer-features.txt
        # another command
      shell: bash
```